src/test/java/edu/cornell/kfs/concur/batch/service/impl/ConcurRequestExtractFileValidationServiceImplTest.java

```diff
@@ -0,0 +1,113 @@
+import edu.cornell.kfs.concur.batch.businessobject.ConcurRequestExtractRequestDetailFileLine;
+import edu.cornell.kfs.concur.batch.businessobject.ConcurRequestExtractRequestEntryDetailFileLine;
+
+public class ConcurRequestExtractFileValidationServiceImplTest {
+    private ConcurRequestExtractFileValidationServiceImpl concurRequestExtractFileValidationService;
+    
+    @Before
+    public void setUp() throws Exception {
+        Logger.getLogger(ConcurRequestExtractFileValidationServiceImpl.class).setLevel(Level.DEBUG);
+        concurRequestExtractFileValidationService = new ConcurRequestExtractFileValidationServiceImpl();
+    }
+    
+    @After
+    public void tearDown() throws Exception {
+        concurRequestExtractFileValidationService = null;
+    }
+    
+    @Test
+    public void testHeaderAmountMatches() {
+        ConcurRequestExtractFile testFile = buildConcurRequestExtractFile();
+        if (concurRequestExtractFileValidationService.fileApprovedAmountsMatchHeaderApprovedAmount(testFile)) {
+            assertTrue("Valid: Header amount matched sum of row amounts from file.", true);
+        }
+        else {
+            assertTrue("Invalid: Header amount did NOT match sum of row amounts from file.", false);
+        } 
+    }
+    
+    @Test
+    public void testHeaderAmountDoesNotMatch() {
+        ConcurRequestExtractFile testFile = buildConcurRequestExtractFile();
+        testFile.setTotalApprovedAmount(new KualiDecimal(9.87));
+        if (concurRequestExtractFileValidationService.fileApprovedAmountsMatchHeaderApprovedAmount(testFile)) {
+            assertTrue("Invalid: Header amount matched row sum. They should NOT have matched.", false);
+        }
+        else {
+            assertTrue("Valid: Header amount did NOT match sum of row amounts from file.", true);
+        } 
+    }
+    
+    @Test
+    public void testHeaderRowCountMatches() {
+        ConcurRequestExtractFile testFile = buildConcurRequestExtractFile();
+        if (concurRequestExtractFileValidationService.fileRowCountMatchesHeaderRowCount(testFile)) {
+            assertTrue("Valid: Header row count file row count.", true);
+        }
+        else {
+            assertTrue("Invalid: Header should NOT have matched file row count", false);
+        } 
+    }
+    
+    @Test
+    public void testHeaderRowCountDoesNotMatch() {
+        ConcurRequestExtractFile testFile = buildConcurRequestExtractFile();
+        testFile.setRecordCount(new Integer(3));
+        if (concurRequestExtractFileValidationService.fileRowCountMatchesHeaderRowCount(testFile)) {
+            assertTrue("Invalid: Header row count matched file row count. They should NOT have matched.", false);
+        }
+        else {
+            assertTrue("Valid: Header row count did NOT match file row count.", true);
+        } 
+    }
+    
+    protected ConcurRequestExtractFile buildConcurRequestExtractFile() {
```

Comment:
This test was modeled after ConcurStandardAccountingExtractServiceImplTest.java 
Why would I need a fixture and that test not?